### ISSUE TITLE
For the paper "unsupervised Learning or Fast.."

### ISSUE THREAD

**Issue by yoonguusong** [voxelmorph/voxelmorph#531]
**Task** (what are you trying to do/register?)

Hello. I am a ph.D student who is doing for registration.
I have a quite easy question
As you wore in the paper "unsupervised Learning for Fast Probabilistic Diffeomorphic Registration", 
there is a Loss function that is applied KL divergence, I want to know how the code is applied.
It seems I have a problem finding that code in this github paper.
could you let me know which line, or which function is available for me to check the KL divergence?
Thanks you for your awesome work

**Comment by adalca** [comment]
Sure, there is a vxm.tf.losses.KL class:
https://github.com/voxelmorph/voxelmorph/blob/4a8f5199aa0e7e6f529f9e2cd12f59621ca5c8fa/voxelmorph/tf/losses.py#L247

**Comment by yoonguusong** [comment]
@adalca

Thank you for your comment.
 Is there any code which uses torch? 
Only tensorflow code is available to check the KL divergence?
Sorry.

**Comment by adalca** [comment]
Sorry I missed this -- I dont think there is a KL code in pytorch, but should be easy to translate from tf :)
